docs/dev-setup.md

```diff
@@ -123,13 +123,33 @@ After you have cloned both repositories, your folder structure should look like:
 |-- src/cluster-api (run `tilt up` here)
 ```
 
+Checkout a stable CAPI release, for example:
+
+```bash
+git checkout release-0.3
+```
+
+Install kustomize
+
+```bash
+cluster-api$ make kustomize
+hack/ensure-kustomize.sh
+fetching kustomize@3.9.1
+
+cluster-api$ ./hack/tools/bin/kustomize version
+{Version:kustomize/v3.9.1 GitCommit:7439f1809e5ccd4677ed52be7f98f2ad75122a93 BuildDate:2020-12-29T19:25:35Z GoOs:linux GoArch:amd64}
+```
+
+If the abov installation does not work, install kustomize using a package manager and move the binary to ```hack/tools/bin/kustomize version``` in the cluster-api repo.
```

Comment:
The path to move the binary seems to be incorrect to me. Do we need `version`?

Reply:
fixed